### DISCUSSION TITLE
Error {"message":"Missing/invalid DB name","status":500} when accessing app directly through its URL

### DISCUSSION THREAD

**Discussion by xlog-null** [Budibase/budibase#8300] [Help] [answered]
Hi, I'm getting this message `{"message":"Missing/invalid DB name","status":500}`  when accessing my app through its direct such as `https://<mydomain>/apps/<myapp>`.

When accessing just `https://mydomain/`, it works fine and then users can access assigned apps through there.

I am using the latest `docker-compose.yaml` config file from the Budibase docs. I am running the latest version of Budibase. 

The thing I've noticed is that my `root_couch-init_1` container crashes on startup, every time with Exit 2 code: 
```
# docker-compose ps -a
          Name                         Command                  State                              Ports
--------------------------------------------------------------------------------------------------------------------------------
bbapps                      docker-entrypoint.sh ./doc ...   Up             4001/tcp
bbproxy                     /docker-entrypoint.sh ngin ...   Up             0.0.0.0:10000->10000/tcp,:::10000->10000/tcp, 80/tcp
bbworker                    docker-entrypoint.sh ./doc ...   Up             4001/tcp
root_couch-init_1           /entrypoint.sh sh -c sleep ...   Exit 2
root_couchdb-service_1      /docker-entrypoint.sh /opt ...   Up             4369/tcp, 5984/tcp, 9100/tcp
root_minio-service_1        /usr/bin/docker-entrypoint ...   Up (healthy)   9000/tcp
root_redis-service_1        docker-entrypoint.sh redis ...   Up             6379/tcp
root_watchtower-service_1   /watchtower --debug --http ...   Up             8080/tcp
```

Running `docker logs 288e60af163b` (the `root_couch-init_1` container logs)`:

```
root@ehl-droplet:~# docker logs 288e60af163b
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (7) Failed to connect to couchdb-service port 5984 after 6 ms: Connection refused
sh: fg: line 1: No current job
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
{"error":"file_exists","reason":"The database could not be created, the file already exists."}
100    95  100    95    0     0    175      0 --:--:-- --:--:-- --:--:--   174
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
```
I don't even know which database it's talking about, and I cannot check as it crashes on start so I can't go into it. And I am not sure if this is the reason for my original problem.

Any help please. I think I am cursed! Thank you

**Comment by melohagan** [comment, marked as answer]
Hey @xlog-null 

Firstly don't worry about couch-init being offline. It only spins up to initialise the couchdb service and is supposed to exit.

It may be possible that your couchdb instance has got into a conflicted state. Could you try following these steps to see if there is a conflict?
https://github.com/Budibase/budibase/discussions/7777#discussioncomment-3732390

Otherwise I would do an app export and backup the data. Then you can import it as a new app and see if the new app has the same issue.

**Reply by xlog-null** [reply]
That was exactly it, thank you.

**Reply by melohagan** [reply]
Awesome, glad that sorted it :)
